build.xml

```diff
@@ -1,126 +1,417 @@
+            <arg value="-config"/>
+            <arg value="${basedir}/.settings/org.eclipse.jdt.core.prefs"/>
+            <arg line="${javafiles}"/>
+        </exec>
+        <travis target="eclipseformat" />
+    </target>
+
+    <target name="eclipseformat-check" depends="eclipseformat">
+        <travis target="eclipseformat-check" start="Check Eclipse Format" />
+        <exec executable="git" dir="${basedir}">
+            <arg value="status" />
+            <arg value="*.java" />
+        </exec>
+        <exec executable="git" dir="${basedir}" failonerror="true">
+            <arg value="diff-index" />
+            <arg value="--quiet" />
+            <arg value="--exit-code" />
+            <arg value="--ignore-submodules" />
+            <arg value="HEAD" />
+        </exec>
+        <travis target="eclipseformat-check" />
     </target>
 
+<!--    <target name="clean">-->
+<!--        <delete dir="${build.dir}"/>-->
+<!--    </target>-->
+
+<!--    <target name="eclipseformat">-->
+<!--      <pathconvert pathsep=" " property="javafiles">-->
+<!--        <fileset dir="${src.dir}">-->
+<!--          <include name="**/*.java"/>-->
+<!--        </fileset>-->
+<!--        <fileset dir="${test.dir}">-->
+<!--          <include name="**/*.java"/>-->
+<!--        </fileset>-->
+<!--      </pathconvert>-->
+<!--      <exec executable="${env.ECLIPSE_EXE}" dir="${basedir}">-->
+<!--          <arg value="-nosplash"/>-->
+<!--          <arg value="-application"/>-->
+<!--          <arg value="-consolelog"/>-->
+<!--          <arg value="-data"/>-->
+<!--          <arg value="${basedir}"/>-->
+<!--          <arg value="-vm"/>-->
+<!--          <arg value="${env.JAVA_HOME}/bin/java"/>-->
+<!--          <arg value="org.eclipse.jdt.core.JavaCodeFormatter"/>-->
+<!--          <arg value="-config"/>-->
+<!--          <arg value="${basedir}/.settings/org.eclipse.jdt.core.prefs"/>-->
+<!--          <arg line="${javafiles}"/>-->
+<!--      </exec>-->
+<!--      <exec executable="git" dir="${basedir}" failonerror="true">-->
+<!--          <arg value="status" />-->
+<!--          <arg value="*.java" />-->
+<!--      </exec>-->
+<!--      <exec executable="git" dir="${basedir}" failonerror="true">-->
+<!--          <arg value="diff-index" />-->
+<!--          <arg value="&#45;&#45;quiet" />-->
+<!--          <arg value="HEAD" />-->
+<!--      </exec>-->
+<!--    </target>-->
+
     <target name="checkstyle-jar">
-        <mkdir dir="${lib.dir}" />
+        <travis target="checkstyle-jar" start="Get Checkstyle Jar" />
         <get src="https://github.com/checkstyle/checkstyle/releases/download/checkstyle-${checkstyle.version}/checkstyle-${checkstyle.version}-all.jar"
              usetimestamp="true"
              dest="${lib.dir}/checkstyle-${checkstyle.version}-all.jar" />
+        <travis target="checkstyle-jar" />
     </target>
-    
-    <target name="checkstyle" depends="checkstyle-jar" description="Check Code with Checkstyle">
```

Comment:
Is there still a check style target?
Would be good to keep this for CI.

Reply:
It was missing indeed, thanks!